src/Glpi/Features/PlanningEvent.php

```diff
@@ -889,7 +889,7 @@ public function getAlreadyPlannedInformation(array $val)
                  $out .= "&amp;forcetab=" . $itemtype . "$1";
             }
             $out .= "'>";
-            $out .= Html::resume_text($val["name"], 80) . '</a>';
+            $out .= \htmlspecialchars(Html::resume_text($val["name"], 80)) . '</a>';
```

Comment:
`Html::resume_text()` returns a `&nbsp;`, therefore, the encoding should be done in this method directly, in a dedicated PR too, probably.